roles/reproducer/tasks/prepare_networking.yml

```diff
@@ -46,7 +56,46 @@
         enabled: true
         state: started
 
+# Since ci_nmstate relies on the inventory_hostname to find its
+# configuration, we have to bend ansible a bit:
+# it does NOT want to get any jinja2 in a dict *key*, meaning we
+# can't provide a nice parameter from within the networking-definition.yml
+# environment file using the `cifmw_target_host`.
+# We have, instead, to build the structure as a plain string, where it gets
+# the proper interpretation, and convert that string into YAML.
+# Of course, if anyone provides their own config, we won't override it.
+# As a matter of fact, `set_fact` doesn't override user-provided parameters
+# anyway.
+- name: Set cifmw_ci_nmstate_instance_config if needed
+  when:
+    - cifmw_ci_nmstate_instance_config is undefined
+  vars:
+    _content: |-
+      {{ cifmw_target_host | default('localhost') }}:
+        interfaces:
+          - name: ocppr
+            type: linux-bridge
+            state: up
+            ipv4:
+              enabled: true
+              address:
+                - ip: 172.22.0.1
```

Comment:
So, this needs to be configurable.

Reply:
You can, by just providing the parameter with the same name.
`set_fact` does NOT override the parameter, meaning you have full control over its value/content.

So if you build `cifmw_ci_nmstate_instance_config` - it will just work. More over, there's a `when` condition as a safety net to NOT override the parameter.